## Identifying a 3' RNAseq method

So we have these species pairs lakes, Little Quarry, Priest and Paxton, that have benthic and limnetic forms of stickleback. Awesome example of evolutionary parallelism where you have repeated 
evolution of these ecotypes in allopatry. Interestingly, no one has looked to see how gene expression compares across lakes and across ecotypes. So one project stemming out of this fieldwork is to 
look at global gene expression levels across these comparisons, to see if there may be parallel changes in regulatory networks that mediate parallel phenotypic evolution. Pretty exciting!

Despite all of the advances in molecular biology and sequencing, it's still really darn expensive to do conventional RNAseq on dozens (or hundreds) of samples. Plus, most of the time we just want 
gene counts i.e. how much of a certain gene is expressed. Bulk RNAseq gets you that, but it is kind of overkill to get the sequence of the entire transcript if all you want to know are the relative 
transcript levels. Enter 3' biased RNAseq methods. These sequence just the 3' end of transcripts, in a so-called tag. This is usually the 3' UTR and maybe a bit of the coding sequence (depending on 
the length of inserts in your libraries and the length of sequencing read). There are a few of them out there, but the first one I thought of was Tag-Seq. Developed by [Meyer et al 
2011](https://onlinelibrary.wiley.com/doi/10.1111/j.1365-294X.2011.05205.x) and refined 
by [Lohman et al 2016](https://onlinelibrary.wiley.com/doi/full/10.1111/1755-0998.12529), this approach fragments RNA by sonication or a magnesium treatment, then does a bunch of molecular magic to 
prepare sequencing libraries with just the 3' end of transcripts. Sounds great, and seems to work great too. A few cores and a commercial service actually offer this service, but it's much more 
expensive than doing yourself. But, the approach is kind of older, and it hasn't been optimized for sequencing on newer platforms (e.g. NovaSeq X). I emailed one of the authors on the Lohman paper, 
and he actually suggested going with [TM3'seq](https://academic.oup.com/g3journal/article/10/1/143/6020296), a tagmentation based approach that is cheaper and easier than TagSeq. Cool! After reading 
and thinking about this a while, it seemed like it would take a while to optimize, given that most labs using it appear to use a homemade version of the Tn5 transposase. It's available commercially, 
but it's potentcy is super variable so we would have to fine-tune it. If only there were something more off the shelf....

Enter Alithea genomics. 
![alithea](https://www.biomol.com/media/image/51/cd/58/Alithea_Genomics_Logo_200x200.png) 

Dan Bolnick suggested we try their BRB-seq kit, which is really similar to TM3'seq but does early multiplexing, right after reverse transcription. This means that all of the 
subsequent steps can be done on pools, vastly simplifying and speeding up the process. He said they had great results with it. I contacted Alithea (based in Switzerland) and had a back and forth 
with a representative. They offer in 96- and 384-well format, with the 384-well intended for working with pipetting robots. A good friend Ariel down the road at JCVI said I may be able to use their 
robot and thermocycler with the right block. Right on! This should make it wayyyy easier. I think because you are doing early multiplexing, right after RT, it's super important that the input mass 
of RNA to each reaction is really even. So I am going to normalize my input RNA using the robot, and then do RT and pooling on it. Then the subsequent steps can be done here at UCSD. 

So, this is a lot of what I have been up to since I returned from Canada on the 16th. I know, maybe I should be doing more than just thinking about RNA. And I have! I'm steadily working my way 
through the 500+ fish and fin clipping them for the genomic analyses. I've also started to brainstorm ideas for NSF PRFB. I'd obviously like to pursue a question in stickleback, but it's not clear 
how distinct from the lab's current funding has to be. I could easily write a proposal just about parallelism, but Diana already has a grant to do exactly that, so not sure if it is novel enough. 
For a while now I have been interested in this idea of mitonuclear coevolution, and I think stickleback could be an exciting system to investigate it. Seems like there is at least one other lab 
doing that (Bill Cresko at U of O). I'd also like to bring some of my physiology expertise into this proposal. There has been a fair amount of therma tolerance type stuff in stickleback (see 
Patricia Schulte's lab), but I think there are potential avenues for integrating these two themes. Something along the lines of what Ron Burton has been looking into in Tigriopus with hybrid 
breakdown and mitonuclear incompatibilities. I dunno. Still early days. We've thought about how epistasis may constrain parallelism in stickleback, and looking at epistatic interactions between the 
nuclear and mitochondrial genome could be an exciting avenue, especially in the context of mitochondrial physiology. Still early days, but I think there could be something there.

Reading about mitochondrial haplotypes I learned that there are two main mitotypes in the Pacific, TNP and ENA. TNP is dominant in west pacific, ENA in east pacific, but there is a zone in alaska 
where frequencies are intermediate. There are two lakes right next to each other in Alaska (Barabara and Sportfishing Lakes) where the populations appear to be fixed for alternative mitotypes. I 
think this could be a really interesting system for looking at mitonuclear coevolution, making crosses and backcrosses to put different mitochondrial genomes on mixed up nuclear genomes with more or 
less ancestry of the co-evolved lake. Hmmmm... Maybe this is going somewhere.

In other new, the RNA extraction kit finally arrived! I just prepped all of the buffers today and will proceed with a couple of test extractions tomorrow. Exciting!

Okay, all for now.  


